rules/linux/persistence_user_password_change.toml

```diff
@@ -0,0 +1,89 @@
+[metadata]
+creation_date = "2024/06/03"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/06/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for Linux user account password changes. Threat actors may attempt to change the password of a user
+account to maintain access to a system.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.file*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux User Account Password Change"
```

Comment:
if the file `/etc/shadow` is modified, does this mean 100% pwd change or it could be also triggered by other action ? maybe adjust the rule name and description to the events/facts (e.g. shadow file mod by unusual process or just shadow filemod).